redis/connector.go

```diff
@@ -40,3 +43,45 @@ func New(uri string) (Client, error) {
 	}
 	return r, nil
 }
+
+// NewSecure - constructor to create an instance of the client
+func NewSecure(uri string, clientCert []byte, clientKey []byte) (Client, error) {
+	l, _ := log.New("")
+	var r Client
+
+	tlsCert, tlsErr := tls.X509KeyPair(clientCert, clientKey)
+	if tlsErr != nil {
+		fmt.Println("err", tlsErr)
+		return r, tlsErr
+	}
+
+	opt, err := redis.ParseURL(uri)
+	if err != nil {
+		l.Error("REDIS_PARSE_URL", err)
+		return r, err
+	}
+
+	// tlsCerts generated with tls.X509KeyPair()
+	// enable TLS connections if input provided
+	if tlsCert.Certificate != nil {
+		opt.TLSConfig = &tls.Config{
+			Certificates: []tls.Certificate{tlsCert},
+			// uncomment below when running on local
```

Comment:
For this maybe can just give a option in the function params that defaults to False
`func NewSecure(uri string, clientCert []byte, clientKey []byte, insecureSkipVerify bool) (Client, error)`

`InsecureSkipVerify: insecureSkipVerify`